src/App.jsx

```diff
@@ -1,107 +1,22 @@
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const postsGlobal = postsFromServer.map(post => (
```

Comment:
```suggestion
const preparedPosts = postsFromServer.map(post => (
```
Let's rename the variable with more meaningful name